falcon/media/urlencoded.py

```diff
@@ -9,7 +10,11 @@ class URLEncodedFormHandler(BaseHandler):
     URL-encoded form data handler.
 
     This handler parses ``application/x-www-form-urlencoded`` HTML forms to a
-    ``dict`` in a similar way that URL query parameters are parsed.
+    ``dict`` in a similar way that URL query parameters are parsed. An empty body
+    will be parsed as an empty dict.
+
+    This handler will raise :class:`falcon.MediaMalformedError` in case of errors,
+    usually only if the body cannot be parsed as ASCII.
```

Comment:
Could it be worth to clarify:
> _When deserializing,_ this handler will...

In addition, not a big deal, but we could probably be slightly more specific than that _usually_ here.
AFAICT there are only two cases where this might raise:
* The request payload is not ASCII as you describe it here.
* Any of the URL-encoded strings is not valid UTF-8.